### ISSUE TITLE
Fix dependabot issue

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1196]
We've had a dependabot alert on the project for a while now that we should try and fix.

**Comment by daveverwer** [comment]
This is tricky to fix without major disruption to our front-end deployment process. When the alert first appeared, I checked how we could upgrade. Unfortunately, that dependency is inside the `rollup-plugin-styles` plugin we use to generate our CSS. I just checked again and it's still not updated.

Worse, that dependency is old. it depends on `normalize-url` version `3.0.0` which was released on the 27th May 2018. Maybe this security issue will be what prompts them to upgrade that dependency, but there is not currently even a GitHub issue open for it, so I'm not holding my breath.

We would need to replace rollup with something different to fix this issue, which I am not keen to do.

I think the next step would be to figure out if we are vulnerable. My gut feeling says we won't be as this is just part of our build process, but we'd need to check more deeply into what the vulnerability is.
